package.json

```diff
@@ -79,6 +79,7 @@
     "qrcode.react": "1.0.1",
     "react": "17.0.2",
     "react-apexcharts": "1.3.9",
+    "react-currency-flags": "0.1.2",
```

Comment:
@Betree : Hope it's okay to use this library. Alternative is to use the native html emojis like we do in, 

https://github.com/opencollective/opencollective-frontend/blob/07ea8a7f5055ddc0ab71d407bc33f642e3e24254/lib/constants/locales.js#L12-L29

But somehow I felt this way is better since now we are using these in multiple places. If agreed I'll convert the `locales.js` to use `react-currency-flags` as well.

Reply:
@SudharakaP What is the impact on the bundle size?

Reply:
Sorry I forgot this part, so it seems that it has a bundle size of 150kb. 🤔 

https://bundlephobia.com/package/react-currency-flags@0.1.2

This does have some impact for sure, but I am a bit hesitant on using the native emojis, since then we have to have some function ourselves to convert the currency codes to respective flag uni-codes. Also since this library uses `transferwise/currency-flags` [under the hood](https://github.com/josephchang/react-currency-flags#readme), we are most certain that flags exist for all the currency codes we specify. But if you think the bundle size don't do justice I am okay with going with the native emoji approach. WDYT? 🤔

Reply:
108.2kB gzipped for a feature that only a few are going to use is **huge**. I'm really not fan of adding something that big to the page, and if you look at https://github.com/transferwise/currency-flags you'll notice that Transferwise themselves got away from this base64 approach - certainly because it's super heavy.

We only have a limited amount of currencies to support, so the emoji approach sounds like a good first MVP to me - even though it has some downsides, mostly the fact that they'll look differently on different machines. Feels like re-inventing the wheel? Maybe that's a good candidate for an independent NPM package :-)

No blocker for me if you prefer to take the image approach, but then I would suggest self-hosting them.

Reply:
Makes sense. I've created a small NPM package for this since I didn't find anything nice to do this. 🙂 

https://github.com/opencollective/country-currency-emoji-flags

I just used only native js for this so the bundle size is around 30kb; https://bundlephobia.com/package/country-currency-emoji-flags@1.0.2. Update this PR to include this new package. Let me know what you think.

Reply:
That's way better! In https://github.com/opencollective/country-currency-emoji-flags/blob/main/country-data.json & https://github.com/opencollective/country-currency-emoji-flags/blob/main/currency-data.json I would suggest:
- Using maps rather than arrays (e.g. `{ EUR: '🇪🇺', GBP: '🇬🇧' }`). We have a unique index, let's use it. Accessing a map is way more performant than filtering a list.
- Having the name and countries may be a bit overkill. We don't need it for now, so I would optimize for bundle size.

Reply:
@Betree : Done. I tried to support some things I thought maybe other or us in the future will find useful. 😉 Anyways I've removed those part and now the package only supports returning emoji flags for given currency codes. The bundle size is reduced to around 9kb. Also used the map instead of the emoji list. 

https://bundlephobia.com/package/country-currency-emoji-flags@1.0.4

Reply:
It's better. Again, I don't think we need the `currencyName`, a simple map of currency <> flag would be enough but that's not a blocker. 2.5kb gzipped is perfectly fine.